R/matrix_form.R

```diff
@@ -7,6 +7,7 @@
 #' @rdname MatrixPrintForm
 #' @aliases MatrixPrintForm-class
 #' @exportClass MatrixPrintForm
+#' @keyword internal
```

Comment:
This constructor is explicitly designed to be used by other packages (e.g., rtables, rlistings), and thus can never be made internal.